examples/redis/README.md

```diff
@@ -0,0 +1,247 @@
+# Redis Destination Example
```

Comment:
Worth rm the example dir? Since we're not actively supporting it outside of our use case?